tests/integration/test_model.py

```diff
@@ -924,3 +924,79 @@ def test_experiment_automl_regressor(c, client, training_df):
     ), "Best model was not registered"
 
     check_trained_model(c, "my_automl_exp2")
+
+
+# TODO - many ML tests fail on clusters without sklearn - can we avoid this?
+@skip_if_external_scheduler
```

Comment:
@charlesbluca Wanted to ask you about this since we have it for several of the `test_model` pytests

Reply:
Opened an issue for this here: https://github.com/dask-contrib/dask-sql/issues/861